source/ShipInfoPanel.cpp

```diff
@@ -570,8 +570,9 @@ void ShipInfoPanel::DrawCargo(const Rectangle &bounds)
 	Color backColor = *GameData::Colors().Get("faint");
 	const Ship &ship = **shipIt;
 
-	// Cargo list.
-	const CargoHold &cargo = (player.Cargo().Used() ? player.Cargo() : ship.Cargo());
+	// Cargo list: Show pooled cargo instead if the ship to display is landed together with the flagship.
```

Comment:
Lowercase after a colon.
```suggestion
	// Cargo list: show pooled cargo instead if the ship to display is landed together with the flagship.
```